.clang-format

```diff
@@ -28,8 +34,8 @@ AllowShortLoopsOnASingleLine: false
 # in this project
 BasedOnStyle: Google
 
-# Increase the column limit to 100
-ColumnLimit: 100
+# Increase the column limit to 101
+ColumnLimit: 101
```

Comment:
It looks like we can set this to 0 and it won't change the line length from what you type.  This should make developing good looking code faster than having to tweak this term.

Reply:
I do want to have a line length limit here. Sorry for this small tweak, it was not necessary, I just like this number better. Do you want me to revert it?

Reply:
No, it's fine. I figured doing line length manually would just save us from doing tweaks like this every few commits. We can keep the limit, if that works better for you.

Reply:
Thanks for your take on this, David. I do want to enforce a line limit since it can easily get out of control with super long lines. The default value for the Google style guide used in OpenTurbine is 80, which is kind of short. I just wanted to find something that's larger than 80 but not too large that it doesn't fit on standard monitors developers work on.